### ISSUE TITLE
Add installation and example for aws ack elastic cache 

### ISSUE THREAD

**Issue by srikanthmalyala** [aws-controllers-k8s/community#1496]
** https://github.com/aws-controllers-k8s/elasticache-controller**

**Comment by A-Hilaly** [comment]
Hi @srikanthmalyala we have documented the steps for installing an ACK controller, please refer to https://aws-controllers-k8s.github.io/community/docs/user-docs/install/

**Comment by srikanthmalyala** [comment]
Yes I see it, thank you but I didn’t find any document for elastic cache CRD resource examples.

**Comment by A-Hilaly** [comment]
Sadly we do not have any examples regarding elasticache CRDs in https://github.com/aws-controllers-k8s/examples/tree/main/resources. However you can always check the samples used in elasticache e2e tests like https://github.com/aws-controllers-k8s/elasticache-controller/blob/main/test/e2e/snapshot/e2e.sh#L52-L73.
/cc @aws-controllers-k8s/elasticache-maintainer

**Comment by A-Hilaly** [comment]
Or even better you can take a look at the ACK generated API Reference docs https://aws-controllers-k8s.github.io/community/reference/elasticache/v1alpha1/cacheparametergroup/ :)
